imagenet/model_saver_cpu.py

```diff
@@ -6,4 +6,4 @@
 
 batch = torch.randn((1, 3, 224, 224))
 traced_model = torch.jit.trace(model, batch)
-torch.jit.save(traced_model, 'resnet50.pt')
+torch.jit.save(traced_model, 'resnet50.CPU.pt')
```

Comment:
Can't we delete this file entirely since model_saver.py now supports both CPU and GPU?

Reply:
Yeah, I think you're right